ci/azure/linux_script

```diff
@@ -61,6 +61,7 @@ make $JOBS install
 
 # look for formatting errors
 release/bin/zig fmt --check .. || (echo "Please run 'zig fmt' to fix the non-conforming files listed above." && false)
+find . -name  "*.zig" ! -path "*/zig-cache/*" ! -path "*/zig-out/*" ! -path "*/build/*" ! -wholename "*/special/compiler_rt.zig" ! -wholename "*/special/compiler_rt/atomics.zig"  | xargs -n 1 build/zig ast-check
```

Comment:
I excluded some compiler_rt files since stage1 and stage2 don't agree on how `@export` works.

Reply:
instead of this, what about adding `--ast-check` to `zig fmt` since it's already doing the work of finding .zig files and excluding zig-cache?

Reply:
What should I do about the two compiler_rt files that do not currently pass the check due to the `@export` error?
```
error: the first @export parameter must be an identifier
        => @export(@import("compiler_rt/stack_probe.zig").zig_probe_stack, .{
                                                         ^
```